src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -258,6 +267,14 @@ namespace AirflowNetwork {
             Num
         };
 
+        enum class DuctSizeMethod
```

Comment:
Same comment. Is `None` the same as `Invalid` here?

Reply:
Same here.